apps/website/src/components/join-us/ValuesSection.tsx

```
@@ -22,14 +25,32 @@ const values = [
-      <SlideList
-        maxItemsPerSlide={3}
-        className="values-section__values"
-      >
-        {values.map((value) => (
-          <Card key={value.title} {...value} className="values-section__value" />
-        ))}
-      </SlideList>
+      {isMobile ? (
+        <div className="values-section__values--mobile flex flex-col gap-4">
+          {values.map((value) => (
+            <div className="values-card flex flex-row gap-4">
```

Biome (1.9.4)

[error] Missing key property for this element in iterable.

The order of the items may change, and having a key can help React identify which item was moved.
Check the React documentation. 

(lint/correctness/useJsxKeyInIterable)